Networking/Networking/Model/JetpackUser.swift

```diff
@@ -0,0 +1,53 @@
+import Codegen
+import Foundation
+
+/// Information of a WP.com user connected to a site's Jetpack if exists
+public struct JetpackUser: Decodable, GeneratedFakeable, GeneratedCopiable {
+
+    /// Whether the user has connected a WP.com account to the site's Jetpack
+    public let isConnected: Bool
+
+    /// Whether the user is the primary account connected to the site's Jetpack
+    public let isMaster: Bool
```

Comment:
super nit: might be more friendly and easier to understand (or like `isPrimaryUser`/`isPrimaryAccount`)

```suggestion
    public let isPrimary: Bool
```